File: SampleSetup/Output/PHP/SampleProject/Protocol/Body/ApiBodyWithNested.php
<?php
namespace SampleProject\Protocol\Body;

class ApiBodyWithNested {
    /**
     * @var string
     */
    public $Prop1;
    /**
     * @var int|null
     */
    public $Prop2;

    class Nested_1 {
        /**
         * @var string
         */
        public $Prop_1_1;
        /**
         * @var int|null
         */
        public $Prop_1_2;

        class Nested_1_2 {
            /**
             * @var string
             */
            public $Prop_1_2_1;
            /**
             * @var int|null
             */
            public $Prop_1_2_2;

    public function __construct()
    {
        $this->Prop_1_2_1 = "";
        $this->Prop_1_2_2 = null;
    }
        }

    public function __construct()
    {
        $this->Prop_1_1 = "";
        $this->Prop_1_2 = null;
    }
    }

    public function __construct()
    {
        $this->Prop1 = "";
        $this->Prop2 = null;
    }
}

File: SampleSetup/Output/Kotlin/SampleProject/Protocol/Body/ApiBodyWithNested.kt
package SampleProject.Protocol.Body

open class ApiBodyWithNested {
    var Prop1: String = ""
    var Prop2: Int? = null

    open class Nested_1 {
        var Prop_1_1: String = ""
        var Prop_1_2: Int? = null

        open class Nested_1_2 {
            var Prop_1_2_1: String = ""
            var Prop_1_2_2: Int? = null
        }
    }
}
